### ISSUE TITLE
Menu - Let's make visual indications or controls that indicate sub-menus are available on the default menu style (Severity 8)

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#602]
**Describe the bug**
https://stanford.levelaccess.net/public/reporting/view_instance.php?instance_id=1065636891

See ticket THEME-17: https://stanfordits.atlassian.net/browse/THEME-17

ID
1065636891
Violation
Ensure sub-menu items are keyboard accessible and communicate sub menu structure
Module Name
Multi Menu
Description
[Issue] 
The 'Default Styling' menu does not have any visual indications or controls that sub-menus are available. A keyboard user will tab through the main navigation and be completely unaware that sub-menu structures exist.

It also does not include any notification to screen reader users that there are sub-menus available when navigating. Without a mouse, they are unable to navigate into these sub-menus.

[User Impact]
When menus do not implement these accessibility features users with disabilities such as users who are blind may not know that a menu has opened or is available. Users who rely on the keyboard may not be able to interact with menu or sub-menu items, etc.

[Code Reference]
<ul class="su-multi-menu__menu-lv1 su-multi-menu__menu">
   <li class="su-multi-menu__item"><a href="/" class="su-multi-menu__link">Home</a></li>
   <li class="su-multi-menu__item su-multi-menu__item--parent">
      <a href="#" class="su-multi-menu__link">Useful Websites</a>
      <ul class="su-multi-menu__menu-lv2 su-multi-menu__menu">
         <li class="su-multi-menu__item"><a href="https://www.stanford.edu" class="su-multi-menu__link" target="_blank">Stanford</a></li>
         <li class="su-multi-menu__item"><a href="https://uit.stanford.edu/sws" class="su-multi-menu__link" target="_blank">Stanford Web Services</a></li>
         <li class="su-multi-menu__item"><a href="https://ucomm.stanford.edu/" class="su-multi-menu__link">University Communications</a></li>
         <li class="su-multi-menu__item"><a href="/section-components.html" class="su-multi-menu__link">Decanter components</a></li>
      </ul>
   </li>
   [///]
   </li>
</ul>
Note
 [Recommendation] 
The other styling menus below it provide more than sufficient information that drop-down menus exist. Advised making one of them the new default.
Line Number
-1
Instance Severity
8

 !menu-indicate-submenus.png|thumbnail! 

**To Reproduce**

**Expected behavior**

**Screenshots**
https://stanfordits.atlassian.net/secure/attachment/40026/menu-indicate-submenus.png
https://stanfordits.atlassian.net/secure/attachment/40102/Screen%20Shot%202020-01-07%20at%2010.59.04%20AM.png

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by kerri-augenstein** [comment]
@cjwest does this resolve the issue that Decanter did not have the drop-down arrows by default, when hover menus were present? It's hard to tell what has been resolved here. Thx!
